### PR TITLE
docs: update Examples & Tutorials page

### PR DESCRIPTION
**Pull request by igorlukanin**
Numerous changes bringing this page up-to-date with the current state of things.

### CODE REVIEW

**Comment by igorlukanin** [comment]
Please review the updated page here: https://deploy-preview-3179--cubejs-docs-staging.netlify.app/examples

**Review by hassankhan — APPROVED** [review]
Looks much better 🙌

File: docs/content/Examples.md [hunk 1]
```diff
@@ -6,86 +6,77 @@ redirect_from:
...
+| [Data blending](https://cube.dev/docs/data-blending)                                     | [Introducing data blending API](https://cube.dev/blog/introducing-data-blending-api/)                                | [Demo](https://data-blending-demo.cube.dev)       |
+| [Real-time data fetch](https://cube.dev/docs/real-time-data-fetch)                       | [Real-time dashboard guide](https://real-time-dashboard.cube.dev)                                                    | [Demo](https://real-time-dashboard-demo.cube.dev) |
+| [Dynamic schema creation](https://cube.dev/docs/dynamic-schema-creation)                 | [Using asyncModule to generate schemas](https://github.com/cube-js/cube.js/tree/master/examples/async-module-simple) | —                                                 |
+| [JSON Web Ket Sets support](http://localhost:8000/security#using-json-web-key-sets-jwks) | [Auth0 integration](https://github.com/cube-js/cube.js/tree/master/examples/auth0)                                   | —                                                 |
```
Comment (hassankhan) [inline docs/content/Examples.md:52]: Typo:

```suggestion
| [JSON Web Key Sets support](http://localhost:8000/security#using-json-web-key-sets-jwks) | [Auth0 integration](https://github.com/cube-js/cube.js/tree/master/examples/auth0)                                   | —                                                 |
```

Also wondering if the feature name should be Authentication, to match the docs? /cc @keydunov

File: docs/content/Examples.md [hunk 2]
```diff
@@ -6,86 +6,77 @@ redirect_from:
...
+| [JSON Web Ket Sets support](http://localhost:8000/security#using-json-web-key-sets-jwks) | [AWS Cognito integration](https://github.com/cube-js/cube.js/tree/master/examples/cognito)                           | —                                                 |
```
Comment (hassankhan) [inline docs/content/Examples.md:53]: Ditto

**Review by hassankhan — CHANGES_REQUESTED** [review]

**Comment by igorlukanin** [comment]
@hassankhan WDYT now?

File: docs/content/Examples.md [hunk 3]
```diff
@@ -6,86 +6,77 @@ redirect_from:
...
+<!-- Also, please review the [recipes](#recipes) for common use cases. -->
+
 
 ## Tutorials
 
-### Getting Started Tutorials
+These tutorials are great to start learning Cube.js:
+
+* [Cube.js, the Open Source Dashboard Framework: Ultimate Guide](https://cube.dev/blog/cubejs-open-source-dashboard-framework-ultimate-guide) —
+  you will launch an API with Cube.js and build a dashboard with React
```
Comment (hassankhan) [inline docs/content/Examples.md:18]: @igorlukanin WDYT about removing all the 'you will' bits from the following bullet points?

```suggestion
  launch an API with Cube.js and build a dashboard with React
```

**Review by hassankhan — APPROVED** [review]
One last minor comment, other than that looks great, a huge improvement 🎉

File: docs/content/Examples.md (hunk 3 above)
Comment (igorlukanin) [reply docs/content/Examples.md]: Great suggestion, fixed!

File: docs/content/Examples.md [hunk 4]
```diff
@@ -6,86 +6,77 @@ redirect_from:
...
-[code-auth0]: https://github.com/cube-js/cube.js/tree/master/examples/auth0
-[code-cognito]: https://github.com/cube-js/cube.js/tree/master/examples/cognito
+Below you can find tutorials to help you get started with Cube.js.
+<!-- Also, please review the [recipes](#recipes) for common use cases. -->
+
 
 ## Tutorials
 
-### Getting Started Tutorials
+These tutorials are great to start learning Cube.js:
```
Comment (rpaik) [inline docs/content/Examples.md:15]:
```suggestion
These tutorials are great places to start learning Cube.js:
```

**Comment by rpaik** [comment]
Looks great @igorlukanin 🚀 Found one minor typo.

**Pull request merged**